### PR TITLE
Swagger docs interactive

### PR DESCRIPTION
**Pull request by BrandonMurnane**
Changed swagger docs so it will use whatever port it is on rather than giving it one

### CODE REVIEW

**Comment by BrandonMurnane** [comment]
@iamigo its the same as trust uses i originally set it up there. Is there an easy way to create review apps ?

**Comment by BrandonMurnane** [comment]
also do you know why the bulk upset test breaks regularly @iamigo

File: api/v1/swagger.yaml [hunk 1]
```diff
@@ -41,8 +42,8 @@ basePath: /v1
 
 # =============================================================================
 schemes:
-  - https
   - http
+  - https
```
Comment (pallavi2209) [inline api/v1/swagger.yaml]: @BrandonMurnane curious to know why this change? Does this change the "try it out feature" to default to http instead of https? Would this have any side effects in terms of api working?

Comment (BrandonMurnane) [inline api/v1/swagger.yaml]: It was a known issue for swagger 2.0 when i originally did this on trust. It means that it will default to http unless the page is served over https in which case it will serve over https. Shouldnt make any difference to how api working just for docs as schemes were set to http and https anyway to allow it to accept both.

**Comment by iamigo** [comment]
@BrandonMurnane holding off on approval and merging until I can see it running in heroku

**Comment by iamigo** [comment]
Brandon why is this PR still open after 25 days? please move it along or close it
